src/Fixer/Phpdoc/AlignMultilineCommentFixer.php

```diff
@@ -0,0 +1,155 @@
+ * @author Julien Falque <julien.falque@gmail.com>
+ */
+final class AlignMultilineCommentFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
+{
+    private $tokenKinds;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $this->tokenKinds = [T_DOC_COMMENT];
+        if ('phpdocs_only' !== $this->configuration['comment_type']) {
+            $this->tokenKinds[] = T_COMMENT;
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Multiline doc comment: enforce asterisk start on each line [PSR-5] and align them.',
+            [
+                new CodeSample(
+'<?php
+    /**
+            * This is a DOC Comment
+with a line not prefixed with asterisk
+
+   */'
+                ),
+                new CodeSample(
+'<?php
+    /*
+            * This is a doc-like multiline comment
+*/',
+                    ['comment_type' => 'phpdocs_like']
+                ),
+                new CodeSample(
+'<?php
+    /*
+            * This is a doc-like multiline comment
+with a line not prefixed with asterisk
+
+   */',
+                    ['comment_type' => 'all_multiline']
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound($this->tokenKinds);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $lineEnding = $this->whitespacesConfig->getLineEnding();
+        foreach ($tokens as $index => $token) {
+            if (!$token->isGivenKind($this->tokenKinds)) {
+                continue;
+            }
+
+            $whitespace = '';
+            $previousIndex = $index - 1;
+            if ($tokens[$previousIndex]->isWhitespace()) {
+                $whitespace = $tokens[$previousIndex]->getContent();
+                --$previousIndex;
+            }
+            if ($tokens[$previousIndex]->isGivenKind(T_OPEN_TAG)) {
+                $whitespace = preg_replace('/\S/', '', $tokens[$previousIndex]->getContent()).$whitespace;
+            }
+
+            if (!preg_match('/\R([ \t]*)$/', $whitespace, $matches)) {
+                continue;
+            }
+
+            $indentation = $matches[1];
+            $lines = preg_split('/\R/', $token->getContent());
+            foreach ($lines as $lineNumber => $line) {
+                if (0 === $lineNumber) {
+                    continue;
+                }
+
+                $line = ltrim($line);
+                if ($token->isGivenKind(T_COMMENT) && (!isset($line[0]) || '*' !== $line[0])) {
+                    if ('all_multiline' !== $this->configuration['comment_type']) {
+                        continue 2;
```

Comment:
Not that would be a valid argument, but just to get know of what others implements:
current projects that use `continue X`:

1. [composer](https://github.com/composer/composer/blob/1.4.2/src/Composer/DependencyResolver/Solver.php#L786)
1. doctrine [ORM](https://github.com/doctrine/doctrine2/blob/v2.5.6/lib/Doctrine/ORM/UnitOfWork.php#L2622) & [DBAL](https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Platforms/SqlitePlatform.php#L1034)
1. [monolog](https://github.com/Seldaek/monolog/blob/1.23.0/src/Monolog/Processor/IntrospectionProcessor.php#L77)
1. [symfony](https://github.com/symfony/symfony/blob/v3.3.2/src/Symfony/Component/Process/Pipes/WindowsPipes.php#L62)
1. [zend framework](https://github.com/zendframework/zend-mail/blob/release-2.8.0/src/Storage/Maildir.php#L73)
1. [phpunit](https://github.com/sebastianbergmann/php-code-coverage/blob/5.2.1/src/Node/File.php#L244)